### ISSUE TITLE
Running throws a local issuer certificate error

### ISSUE THREAD

**Issue by taylorn16** [bitjson/typescript-starter#115]
* **I'm submitting a ...**
[x] bug report
[ ] feature request
[ ] question about the decisions made in the repository
[ ] question about how to use this project

### Summary

I am using node v10.9.0, npm v6.4.0, Windows 10, running command via git bash in my C:\ drive. 

When I run `npx typescript-starter`, the only output I see is this, before the process exits and returns terminal control:

```
npx: installed 357 in 20.282s

  unable to get local issuer certificate
```

I also tried running it from the global cache by doing `npm i -g typescript-starter` and then `typescript-starter` directly. The results from that call are below, which lead me to believe this is not an issue with `npx` itself:

```
  unable to get local issuer certificate
```

#### Other details

* my `npm config get strict-ssl`  is true, but I have a valid, working `cafile` path set. I know the cafile works because if I delete it's contents and try to run any `npm install` command anywhere, it throws an error about the certificate. When I replace its contents and run again, everything works as normal.
* I am using a private NPM registry that mirrors npmjs.com plus my company's internal stuff.
* Turning on verbose logging in npm doesn't seem to do anything, so I'm not sure how to get more details.
* Using npx to run `npx -p typescript tsc --init` successfully creates a tsconfig.json file, so I *really* don't think this is an issue with npx itself

**Comment by bitjson** [comment]
Hi @taylorn16 thanks for the detailed report!

I'm afraid I haven't tested with a private NPM registry or many non-standard networking situations. If you're still interested in debugging, you might want to try dropping some `console.log` statements in here:
https://github.com/bitjson/typescript-starter/blob/8eaf9636c6cfbd9b1b5c0b4393c32f35f06e8d7d/src/cli/tasks.ts#L23-L61

I imagine the issue happens before or when we try to clone this repo from GitHub. If you have anything preventing you from cloning from GitHub, that might be the issue.

**Comment by bitjson** [comment]
Since this has gone a bit stale, I'm going to go ahead and close this issue. 

If anyone is interested in adding support for situations where the local machine can't clone from `github.com`, please comment, and I'll be happy to re-open!

**Comment by rGunti** [comment]
I have found a similar problem when using this at work. My error was `self signed certificate in certificate chain`.
I checked out the code locally and removed the section that checks for updates and I was able to continue. Maybe a switch to disable the update check could resolve this.

Also one needs to make sure that HTTP_PROXY and HTTPS_PROXY environment variables are set correctly for Git to take them up. (In PowerShell on Windows, these are `$env:HTTP_PROXY` and `$env:HTTPS_PROXY`)
